paddle/go/pserver/service.go

```diff
@@ -135,7 +138,10 @@ func (s *Service) SendGrads(grads []Gradient, dummy *int) error {
 	errCh := make(chan error, count)
 	for _, g := range grads {
 		go func(p Parameter, g Gradient) {
-			err := s.opt.UpdateParameter(p, g)
+			opt, err := s.optimizerMap[p.Name]
```

Comment:
sorry, I haven't written the cmake script in right way. I will check in code more carefully. Thanks for your go lint editor plugin and your commit in go cmake script, now I can run this part. Thanks!